modules/bullet/slider_joint_bullet.cpp

```diff
@@ -366,6 +366,7 @@ void SliderJointBullet::set_param(PhysicsServer::SliderJointParam p_param, real_
 		case PhysicsServer::SLIDER_JOINT_ANGULAR_ORTHOGONAL_SOFTNESS: setSoftnessOrthoAng(p_value); break;
 		case PhysicsServer::SLIDER_JOINT_ANGULAR_ORTHOGONAL_RESTITUTION: setRestitutionOrthoAng(p_value); break;
 		case PhysicsServer::SLIDER_JOINT_ANGULAR_ORTHOGONAL_DAMPING: setDampingOrthoAng(p_value); break;
+		case PhysicsServer::SLIDER_JOINT_MAX: break; // Can't happen, but silences warning
```

Comment:
Also here, it's useless check addition

Reply:
The alternative would be `default: {}` but that means that if we do add more enum values, we won't know about them if we forgot to add them in the `switch`, so I prefer to be explicit.

The *real* fix would be to stop using `NAME_MAX` last enum entries to keep track of their number of items, but as it's used in many places in the engine and is relatively convenient, I don't think it would be a good change.